### PR TITLE
improvements/data_structures

### PR DESCRIPTION
**Pull request by Ronald-TR**
Refer #4 
Add:
* data_structures1

### CODE REVIEW

File: pythonlings/i18n/data_structures.en.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+en:
+  data_structures1: |
+    In Python, lists are mutable data structures extremally flexible that we can use to store anything!
```
Comment (marioidival) [inline pythonlings/i18n/data_structures.en.yml]:
```suggestion
    In Python, the lists are mutable data structures extremely flexible that we can use to store anything!
```

File: pythonlings/i18n/data_structures.en.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    Once created, we can add, remove or change their values trough indices.
```
Comment (marioidival) [inline pythonlings/i18n/data_structures.en.yml]:
```suggestion
    Once created, we can add, remove or change their values through indices.
```

**Review by marioidival — CHANGES_REQUESTED** [review]
typo :heart: 

Good job my friend

File: pythonlings/i18n/data_structures.en.yml (hunk 1 above)
Comment (Ronald-TR) [reply pythonlings/i18n/data_structures.en.yml]: Thank you so much!! Done!!  <3

File: pythonlings/i18n/data_structures.en.yml (hunk 2 above)
Comment (Ronald-TR) [reply pythonlings/i18n/data_structures.en.yml]: Done!!  <3

**Pull request merged**